Makefile

```
@@ -182,6 +182,22 @@ open(sys.argv[2], 'w').write(content)" \
 	done
 	@echo "Done."
 
+##@ API Module
+
+API_VERSION ?= $(error API_VERSION is required (e.g. make tag-api API_VERSION=v0.0.2))
+
+.PHONY: tag-api
+tag-api: ## Tag a new release of the api/ Go module (e.g. make tag-api API_VERSION=v0.0.2).
```

checkmake (0.3.2)

[warning] Target body for "tag-api" exceeds allowed length of 5 lines (9).

(maxbodylength)